### ISSUE TITLE
Result service doesn't start

### ISSUE THREAD

**Issue by gvilarino** [dockersamples/example-voting-app#94]
**Description**

The `result` service app, as provided, fails to start and thus the web app is down.

**Steps to reproduce the issue, if relevant:**
1. `git clone https://github.com/dockersamples/example-voting-app.git`
2. `docker-compose up -d`
3. `docker-compose logs -f result`

**Describe the results you received:**
```
result_1  | [nodemon] 1.12.1
result_1  | [nodemon] to restart at any time, enter `rs`
result_1  | [nodemon] watching: *.*
result_1  | [nodemon] starting `node --debug server.js`
result_1  | (node:19) [DEP0062] DeprecationWarning: `node --debug` and `node --debug-brk` are invalid. Please use `node --inspect` or `node --inspect-brk` instead.
result_1  | [nodemon] app crashed - waiting for file changes before starting...
```
**Describe the results you expected:**
```
result_1  | [nodemon] 1.12.1
result_1  | [nodemon] to restart at any time, enter `rs`
result_1  | [nodemon] watching: *.*
result_1  | [nodemon] starting `node server.js`
result_1  | Thu, 23 Nov 2017 17:20:51 GMT body-parser deprecated bodyParser: use individual json/urlencoded middlewares at server.js:67:9
result_1  | Thu, 23 Nov 2017 17:20:51 GMT body-parser deprecated undefined extended: provide extended option at ../node_modules/body-parser/index.js:105:29
result_1  | App running on port 80
result_1  | Connected to db
```

**Additional information you deem important (e.g. issue happens only occasionally):**

The service `command` seems to be faulty.

**Output of `docker version`:**

```
Client:
 Version:      17.11.0-ce-rc4
 API version:  1.34
 Go version:   go1.8.4
 Git commit:   587f1f0
 Built:        Thu Nov 16 01:17:06 2017
 OS/Arch:      darwin/amd64

Server:
 Version:      17.11.0-ce-rc4
 API version:  1.34 (minimum version 1.12)
 Go version:   go1.8.5
 Git commit:   587f1f0
 Built:        Thu Nov 16 01:43:23 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Output of `docker info`:**

```
Containers: 18
 Running: 5
 Paused: 0
 Stopped: 13
Images: 583
Server Version: 17.11.0-ce-rc4
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 992280e8e265f491f7a624ab82f3e238be086e49
runc version: 0351df1c5a66838d0c392b4ac4cf9450de844e2d
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.60-linuxkit-aufs
Operating System: Docker for Mac
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.952GiB
Name: linuxkit-025000000001
ID: HDHC:P4O3:CPUG:QWG7:EMRL:XDGC:76QM:U4JZ:YBLM:TPDY:JMEJ:OFMW
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 63
 Goroutines: 73
 System Time: 2017-11-23T17:32:35.384856415Z
 EventsListeners: 2
No Proxy: *.local, 169.254/16
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, Docker for Mac, Docker for Windows, VirtualBox, physical, etc.):**
Docker for Mac

**Comment by gvilarino** [comment]
Removing the `--debug` flag seems to fix it, although not sure it's the desired intention.

**Comment by dkeightley** [comment]
Can confirm this was an issue for me, and removing --debug worked also

**Comment by ManoMarks** [comment]
Thanks, fixed in #95
